cmd/update-rules/main_test.go

```diff
@@ -163,3 +163,40 @@ func TestUpdateRules(t *testing.T) {
 		})
 	}
 }
+
+func TestDeleteRules(t *testing.T) {
+	tests := []struct {
+		name      string
+		branch    string
+		goVersion string
```

Comment:
We dont need go version for testing deletion. May be can set it outside the struct and use it, as it does not change with every case.

Reply:
we would need go version since its being used in method `UpdateRules(rules, tt.branch, tt.goVersion, true)`

Reply:
I was thinking of setting it outside the struct, but doesnt matter.

Reply:
Will resolve it for now